src/js/chapter-factory.js

```diff
@@ -0,0 +1,48 @@
+import { makeDraggable } from "./drag-drop";
+
+const CreateChapter = (text, img) => {
+  const createButtons = () => {
+    const homeButton = document.createElement("button");
+    const nextChapterButton = document.createElement("button");
+
+    homeButton.textContent = "home";
+    nextChapterButton.textContent = "-->";
+
+    return { homeButton, nextChapterButton };
+  };
+
+  const setupImages = (images) => {
+    let gallery = [];
+
+    images.forEach((image) => {
+      const picture = document.createElement("img");
+      picture.src = image;
+      makeDraggable(picture);
+      gallery.push(picture);
+    });
+
+    return gallery;
+  };
+
+  const createChapterPage = () => {
+    const section = document.createElement("section");
+    const content = document.querySelector("#content");
+    const { homeButton, nextChapterButton } = createButtons();
+    const story = document.createElement("p");
+    const images = setupImages(img);
+
+    content.textContent = "";
```

Comment:
For the createChapterPage, would it be better to just return the created "root" DOM node (in this case, section)? 
I feel like since all page changes need to target #content, it would be better to just have a separate render function specifically for clearing #content and appending a new node to #content.

So in this case, createChapterPage will just return a node that has the content for its page, without needing to know how to render to #content, since that responsibility can be for a separate reusable function. What do you think?